src/Compiler/pars.fsy

```diff
@@ -3406,71 +3406,18 @@ bindingPattern:
   | headBindingPattern
     { $1, rhs parseState 1 }
 
-// Subset of patterns allowed to be used in implicit ctors.
```

Comment:
I guess that's what you're doing in this PR?

Reply:
Exactly. It was really funny to realize it was my comment from 2019. 🙂